app/views/organization_status_types/_form.html.erb

```diff
@@ -2,8 +2,9 @@
   <%= hidden_field(nil, :from_organiation_status, :value => params[:from_organiation_status]) %>
   <div class="form-inputs">
     <%= f.input :name %>
-    <%= f.input :display, :as => :boolean %>
   </div>
+  <%= f.input :display, :as => :boolean %>
+
   <div class="form-actions">
```

Comment:
Not sure, I changed it back and didn't do anything different so I put it back inside the div to keep it consistent